File: FCUBEAPP/src/app/models/sparesstockrptlistmodel.ts
import { Paginationmodel } from "./paginationmodel";
import { Sparesstockrptmodel } from "./sparesstockrptmodel";

export class Sparesstockrptlistmodel {
    sparesStockRptList: Sparesstockrptmodel[] = [];
    pageMetaData: Paginationmodel = new Paginationmodel;
}

File: FCUBEAPI/FleetTrans/Models/SparesStockRptListModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Shared.Models;

namespace FleetTrans.Models
{
    public class SparesStockRptListModel
    {
        public List<SparesStockRptModel> SparesStockRptList { get; set; }
        public PaginationMetaData PageMetaData { get; set; }
    }
}
